webapp/app/js/perf.js

```diff
@@ -110,6 +110,13 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       $scope.$digest();
 
       this.plot.unhighlight();
+
+      //Highlight the points persisted in the url
+      $scope.seriesList.forEach(function(series, i) {
+        if (series.highlighted.length > 0 && series.visibility) {
```

Comment:
Change this to:

if (series.highlighted && ...)

That will ensure old urls still work.

Also, please factor this out into a common function like highlightDatapoints and call it from everywhere you're using, instead of duplicating this block. :)